addons/im_livechat/static/src/messaging/entity/thread/thread.js

```diff
@@ -36,20 +53,11 @@ registerInstancePatchEntity('Thread', 'im_livechat.messaging.entity.Thread', {
      * @override
      */
     _computeDisplayName() {
-        if (this.channel_type === 'livechat') {
-            return this.correspondent_name;
+        if (this.channel_type === 'livechat' && this.correspondent) {
+            return this.correspondent.nameOrDisplayName;
```

Comment:
question: shouldn't we just rely on `correspondent` being set, no matter the channel type? And then code in mail?

Reply:
We could, but I also think it's safe to have rendering per type of thread/channel.

Reply:
ok